code/public/index.html

```diff
@@ -13,7 +13,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title>Popular Movies 🍿🎥</title>
```

Comment:
Nice title! 
Just out of curiosity, we are wondering how you have mounted the favicon-"emoji". We can see it as a file both in the public-folder and in the code-folder, but thought that it would show here in the index.html file also 🤔